.azure/ci.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - '*'
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      endpoint: Planning-Inspectorate
+      name: Planning-Inspectorate/common-pipeline-templates
+      ref: refs/tags/release/3.10.0
+
+extends:
+  template: stages/wrapper_ci.yml@templates
+  parameters:
+    skipAzureAuth: true
+    validationJobs:
+      - name: Run Checks
+        steps:
+          - template: ../steps/node_script.yml
+            parameters:
+              nodeVersion: 18
+              script: npm ci
+          - template: ../steps/node_script.yml
+            parameters:
+              nodeVersion: 18
+              script: npm run check-required
+          - template: ../steps/node_script.yml
+            parameters:
+              nodeVersion: 18
+              script: npm run gen-docs
```

Comment:
We are checking for docs generated, can we only do the same for JS types `npm run gen-types` and also the python types with `python3 pins_data_model/gen_models.py`?

Reply:
`gen-types` and the python models aren't deterministic unfortunately. The ordering of the types seems to change, and python models have a timestamp in them!